test/getNetworkType.js

```diff
@@ -0,0 +1,37 @@
+/*
+    Copyright 2018 The caver-js Authors
```

Comment:
```suggestion
    Copyright 2019 The caver-js Authors
```

Reply:
Please make sure new license statements should include the corresponding year.

Reply:
Oh, okay my bad ;-;! Thank you for reviewing